xprof/sync_daemon_fs

```diff
@@ -47,14 +40,14 @@ def global_barrier(f)
   # Fortunately `count_file` will return -2 for a non exciting folder
   #   hence we test for > 0 and not `!= 0`
   FileUtils.rm_rf(f)
-  busy_wait { count_file(SHARED_GLOBAL_FILESYSTEM) > 0 }
-  FileUtils.rm_rf(SHARED_GLOBAL_FILESYSTEM) if mpi_master?
+  busy_wait { count_file(SHARED_GLOBAL_FILESYSTEM) <= 0 }
```

Comment:
I see you moved `busy_wait` to `iprof`. Does this file really need to load all of `iprof`? Why not have a common that both `xprof.rb.in` and this file require?

Reply:
I thought about that. But then one more file :(  One more `cp` and `makefile` problem to ship in `shared`. So i did the minimal change